docs/_includes/common/listbox-button.html

```diff
@@ -10,7 +10,7 @@ <h3 id="listbox-button-default">Default</h3>
     <div class="demo">
         <div class="demo__inner">
             <span class="listbox-button">
-                <button class="expand-btn" aria-expanded="false" aria-haspopup="listbox">
+                <button class="expand-btn" style="min-width: 150px" aria-expanded="false" aria-haspopup="listbox">
```

Comment:
Do we need the min-width? Just wondering why it has to be added now since it wasn't there before.

Reply:
It's to avoid the button from resizing when its label is updated. I imagine app teams would need to do something similar.